internal/adapters/operation/operation_storage_redis.go

```diff
@@ -365,3 +365,28 @@ func buildOperationFromMap(opMap map[string]string) (*operation.Operation, error
 		ExecutionHistory: executionHistory,
 	}, nil
 }
+
+func (r *redisOperationStorage) CleanExpiredOperations(ctx context.Context, schedulerName string) error {
+	operationsIDs, err := r.client.ZRangeByScore(ctx, r.buildSchedulerHistoryOperationsKey(schedulerName), &redis.ZRangeBy{
+		Min: "-inf",
+		Max: "+inf",
+	}).Result()
+
+	pipe := r.client.Pipeline()
+
+	if len(operationsIDs) == 0 {
```

Comment:
Hey @mflilian are you executing a for loop range on the `operationsIDs` but this `if` ensures that it will always be empty. I think that you are trying to ensure that it is not empty, right?

Reply:
hey, arthur! you're right, just updated this PR fixing it.

Reply:
Hey @mflilian do you already add a test to this case?

Reply:
i updated this case